### DISCUSSION TITLE
安全论坛 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#158] [模板讨论]
安全论坛 模板问题反馈评论区
有问题可以下面反馈。

**Comment by Biqio** [comment]
今天刚注册账号使用，报了这个错。不知是不是网站做了更新

日志：Failed at 1/5 request,
Fail assert: {"re": "\"code\"", "from": "content"} from failed_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.youjiaoku.com/wp-json/jwt-auth/v1/token

**Reply by Biqio** [reply]
问题已解决，是我填反了账号密码，蛋疼

**Reply by wjf0214** [reply]
OK，我才发现，我幼教库漏了，和安全论坛配置成一个地址了。后续可以到幼教库的地址反馈。[幼教库 模板问题反馈评论区](https://github.com/wjf0214/qd-templates/discussions/172)

**Comment by miaoxiaozhao** [comment]
网站已经无法登录，可以去除掉了。

**Reply by wjf0214** [reply]
我再等几天，不确定是网站挂了一阵，还是说已经跑路了。有些站点过几天就恢复了。
